isaric/parsers/isaric-rapid.toml

```diff
@@ -1538,6 +1538,18 @@
   date = { field = "daily_date" }
   value = { field = "daily_temp_vsorres" } # there is no source unit field
 
+[[observation]]
+  name = "fio2_percent"
+  phase = "study"
+  date = { field = "ccm_a_dat" }
+  value = { field = "ccm_a_fio2b_lborres", apply = { function = "Percentage" } }
+
+[[observation]]
+  name = "fio2_percent"
+  phase = "study"
+  date = { field = "ccm_a_dat" }
+  value = { field = "ccm_a_fio2_lborres", apply = { function = "Percentage" } }
+
```

Comment:
same comment for applying "Percentage" function here. 
Fi02 can also be captured in L/min, and an approximate conversion can be made from this to % Fi02, but it is dependent on the method of oxygen therapy. Should we create an fio2_Lmin attribute to capture this, then users can decide for themselves how/if they wish to convert the values?